packages/glsurf/glsurf.3.3.1/opam

```diff
@@ -15,7 +13,7 @@ remove: [
   ["rm" "-rf" "%{doc}%/glsurf"]
 ]
 depends: [
-  "camlimages"
+  "camlimages.all_formats"
```

Comment:
Unfortunately, this field is for *opam* dependencies where `camlimages` is the right value. The place that needs to refer to *only* `camlimages.all_formats` and *not* `camlimages` is inside your build system somewhere. Perhaps you have already fixed that as the archive checksum has changed. If so, reverting this line should be all that is necessary.